PJ05/src/Request/CreateConversationRequest.java

```diff
@@ -1,4 +1,13 @@
 package Request;
 
-public class CreateConversationRequest {
+import java.util.UUID;
+
+public class CreateConversationRequest extends Request {
+    public final UUID user_uuids;
+    public final String name;
+
+    public CreateConversationRequest(UUID user_uuids, String name) {
+        this.user_uuids = user_uuids;
```

Comment:
You need to call super() in order to initiate the Request with a `Request.uuid`. Same for all other requests.

Reply:
Actually `super()` is auto called if it's not specified. My fault.
https://stackoverflow.com/questions/2054022/is-it-unnecessary-to-put-super-in-constructor